File: functions/CronIntegrationPython/handlers/installation_handler.py
from zcatalyst_cliq.installation_handler import (
    handle_installation,
    InstallationRequest,
    InstallationResponse
)


@handle_installation
def installation_handler(req: InstallationRequest, res: InstallationResponse, *args):
    '''Logic for installation post handling'''
    res.status = 200
    return res

File: .build/functions/nodeintegcrontun/handlers/installation-handler.js
'use strict';
import Cliq from 'zcatalyst-integ-cliq';
const installationHandler = Cliq.installationHandler();

installationHandler.handleInstallation(async (req, res) => {
    /* 
    // Logic for installation post handling
    *{
    *
    *}
    */

   res.status = 200;
   return res;
});
